PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/BubbledGraphTest.java

```diff
@@ -0,0 +1,21 @@
+package nl.tudelft.pl2016gr2.model;
+
+public class BubbledGraphTest extends GraphTest {
+  
+  /**
+   * Setup, creates a new graph.
+   */
```

Comment:
`{@inheritDoc}` is only needed for explicit comment inheritance, i.e. if you want to inherit the documentation and want to add extra details to it. For comment inheritance without any changes (exact copy of the parent class' comments) you can just leave out the comment section completely.